signal2html/__main__.py

```diff
@@ -2,9 +2,7 @@
 
 """Command line interface
 
-Author: G.J.J. van den Burg
 License: See LICENSE file.
-Copyright: 2020, G.J.J. van den Burg
```

Comment:
Perhaps the license file should mention '... and contributors'? I don't know if that's typical.

Reply:
Good catch! What do you think about [allcontributors](https://allcontributors.org/)?

Reply:
Sure if you want, I didn't know about that tool.

Reply:
Let's leave it for now then, we can always add it later.